CMakeLists.txt

```diff
@@ -361,12 +362,16 @@ endif()
 
 if(CPACK_GENERATOR STREQUAL "DEB")
```

Comment:
To fix the `DEB; TGZ` issue, this should be changed to check that `CPACK_GENERATOR` contains `DEB`.